docgen/src/guide/Virtual_widgets.md

```diff
@@ -28,6 +29,7 @@ const App = () =>
     <Menu attributeName="fruits" defaultRefinement="Orange" />
   </InstantSearch>;
 ```
+In this case, we are using the `VirtualMenu` to pre refine our results (within the `clothes` search, only display `hoodies`). Think of the `VirtualMenu` as a hidden filter where we define attributes and values that will always apply the our search results.
```

Comment:
I wonder if there is a typo here? Did you mean "define attributes and values that will always apply whenever the page is reloaded"?

Reply:
@marielaures I believe I meant to say:
> ...define attributes and values that will always be applied to our search results.

I really wish I could use a picture here to illustrate it better so there is no confusion at all. I picture a set of checkbox filters where some selections are always selected, on every search, yet the checkbox filters are hidden from the viewer.

Anyhow, does that read better or do you have anything else better to clear it up?

Reply:
Adding a picture is a great idea!